nncf/quantization/algorithms/accuracy_control/ranker.py

```diff
@@ -99,7 +99,8 @@ def find_groups_of_quantizers_to_rank(self, quantized_model_graph: NNCFGraph) ->
         ]
 
         quantized_model_graph_without_shapeof = remove_shapeof_subgraphs(
-            deepcopy(quantized_model_graph), self._algo_backend.get_shapeof_metatypes()
+            deepcopy(quantized_model_graph),
+            self._algo_backend.get_start_nodes_for_activation_path_tracing(quantized_model_graph),
         )
```

Comment:
The resulting graph shouldn't contain so-called `ShapeOf` subgraphs.
The `get_start_nodes_for_activation_pa..` method returns the `model_utils.get_input_nodes` result.
The `get_input_nodes` returns the `nncf_graph.get_input_nodes() + nncf_graph.get_nodes_by_metatypes([OVReadValueMetatype])`.

Where is the `ShapeOf` here in the updated version?

Reply:
That's obvious bug, I'll check why test didn't fail on this.

Reply:
I've tested it by the `test_find_groups_of_quantizers_to_rank`. @andrey-churkin, please take a look